apps/typed-dependency-tester/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "typed-dependency-tester",
+  "private": true,
+  "author": "Microsoft Teams",
+  "description": "A tester to check if types in the dependency array will cause errors",
+  "version": "0.0.1",
+  "scripts": {
+    "build": "pnpm i && pnpm copy && cd ./types && pnpm tsc && cd ../ && pnpm clean",
+    "copy": "mkdir -p types && cp -R ./node_modules/@microsoft/teams-js/dist/esm/packages/teams-js/dts/* ./types || xcopy .\\node_modules\\@microsoft\\teams-js\\dist\\esm\\* .\\types /Y",
+    "clean": "rimraf node_modules && rimraf MicrosoftTeams.d.ts && rimraf ./types"
```

Comment:
Is the `rimraf MicrosoftTeams.d.ts` command necessary anymore if we are only copying over the esm files?

Reply:
good catch, I originally had a `buildNPM` and `buildCDN` command and this was left over from a `cleanCDN` command. Removing it now.